htdocs/contrat/list.php

```diff
@@ -433,6 +437,15 @@
 	$moreforfilter.=$form->selectarray('search_product_category', $cate_arbo, $search_product_category, 1, 0, 0, '', 0, 0, 0, 0, 'maxwidth300', 1);
 	$moreforfilter.='</div>';
 }
+// If the user can view contract type contact
+if ($conf->contrat->enabled && $user->rights->contrat->lire && $user->rights->service->lire)
+{
+	$moreforfilter.='<div class="divsearchfield">';
+	$moreforfilter.=$langs->trans('LinkedToSpecificThirdParty'). ': ';
```

Comment:
Is is to search contract linked to
- a third party
- contact of thirdparty
- or a type of contact of thirdparty ?

Reply:
To search contract linked to a type of contact of thirdparty